tests/ga/test_monitor.py

```diff
@@ -122,13 +138,25 @@ def test_add_sysinfo_should_honor_sysinfo_values_from_agent_for_agent_events(sel
         sysinfo_role_name_value = "sysinfo_dummy_role"
         sysinfo_role_instance_name_value = "sysinfo_dummy_role_instance"
         sysinfo_execution_mode_value = "sysinfo_IAAS"
+        GAVersion_value = "WALinuxAgent-2.2.44"
+        OpcodeName_value = "10101010"
+        EventTid_value = 140240309798656
+        EventPid_value = 108573
```

Comment:
According to the XML example, these two are strings, not ints?

Reply:
In the XML, they are ints - `mt:uint64`. And are casted as ints, when reading the event.
```
    <Param Name="EventTid" Value="140240309798656" T="mt:uint64"/>
    <Param Name="EventPid" Value="108573" T="mt:uint64"/>
```

Reply:
Sorry, I must have been looking at the wrong parameter :)